ql/ql/src/codeql_ql/ast/internal/AstNodes.qll

```diff
@@ -228,7 +226,6 @@ module AstConsistency {
     not exists(node.getParent()) and
     not node.getLocation().getStartColumn() = 1 and // startcolumn = 1 <=> top level in file <=> fine to have no parent
     exists(node.toString()) and // <- there are a few parse errors in "global-data-flow-java-1.ql", this way we filter them out.
-    not node instanceof YAML::YAMLNode and // parents for YAML doens't work
```

Comment:
The `toQL(node)` on line 227 makes this trivially true, so I just dropped this conjunct.